src/mpyl/steps/deploy/k8s/helm.py

```diff
@@ -60,28 +71,86 @@ def __remove_existing_chart(
     )
 
 
-def install(
+def __execute_install_cmd(
     logger: Logger,
-    chart: dict[str, CustomResourceDefinition],
     step_input: Input,
     chart_name: str,
     name_space: str,
     kube_context: str,
     delete_existing: bool = False,
+    additional_args: str = "",
 ) -> Output:
     if delete_existing:
         removed = __remove_existing_chart(logger, chart_name, name_space, kube_context)
         if not removed.success:
             return removed
 
+    cmd = f"helm upgrade -i {chart_name} -n {name_space} --kube-context {kube_context} {additional_args}"
+    if step_input.dry_run:
+        cmd = (
+            f"helm upgrade -i {chart_name} -n namespace --kube-context {kube_context} {additional_args} "
+            f"--debug --dry-run"
+        )
+
+    logger.debug(f"Executing: {cmd}")
+    return custom_check_output(logger, cmd)
+
+
+def install_with_values_yaml(
+    logger: Logger,
+    step_input: Input,
+    values: dict,
+    release_name: str,
+    chart_name: str,
+    namespace: str,
+    kube_context: str,
+) -> Output:
```

Comment:
I'm wondering 🤔 if you could use `write_chart` perhaps? For our standard HELM chart writing logic we already create a `values.yml` file in the usual place, albeit empty.
Perhaps it would be more intuitive to add the possibility to really write values there?

Reply:
i have extending the `write_chart`-argument list by a dict of `values`, for the normal helm-chart-writes we pass an empty dict, and for the dagster-helm-chart, we are passing  an empty chart and the values that need to be written to file